prototype_source/pt2e_quant_x86_inductor.rst

```diff
@@ -134,14 +138,22 @@ quantize the model.
   `multiplications are 7-bit x 8-bit <https://oneapi-src.github.io/oneDNN/dev_guide_int8_computations.html#inputs-of-mixed-type-u8-and-s8>`_. In other words, potential
   numeric saturation and accuracy issue may happen when running on CPU without Vector Neural Network Instruction.
 
+The quantization config is for static quantization by default. To apply dynamic quantization, add an argument ``is_dynamic=True`` when getting the config.
+
+::
```

Comment:
please use code-block directives instead of ::. 
```suggestion
.. code-block:: python
```

Reply:
Thanks.